src/cpp/session/modules/ModuleTools.R

```diff
@@ -410,7 +410,13 @@
    args <- c(c("+RTS", paste0("-K", stack_size), "-RTS"), args)
    
    # additional options
-   args <- c(args, "--self-contained")
+   if (requireNamespace("rmarkdown", quietly = TRUE) && rmarkdown::pandoc_available("2.19")) {
+     args <- c(args, "--embed-resources", "--standalone")
```

Comment:
I think, because RStudio bundles pandoc, we could just use this by default. I think it's very unlikely someone would want to configure RStudio to use an external but older version of pandoc, at least?

Reply:
Agreed! Left just a comment in case someone trips on this.